config/Makefile.j2

```diff
@@ -0,0 +1,119 @@
+TMP_DIR = tmp
+BOOMER_INPUT_DIR = boomer_input
+BOOMER_OUTPUT_DIR = boomer_output
+
+ALL_RUNS={% for run in config.boomer_config.runs %}{{ run.id }} {% endfor %}
+
+$(TMP_DIR)/ {% for run in config.boomer_config.runs %}$(TMP_DIR)/{{ run.id }}/ $(BOOMER_INPUT_DIR)/{{ run.id }}/ $(BOOMER_OUTPUT_DIR)/{{ run.id }}/ {% endfor %}:
+	mkdir -p $@
+
+.PHONY: all
+all: prepare-data boomer
+	make reports # Needs to be run like this, because the jsons are unknown before the boomer run
+
+.PHONY: prepare-data
+prepare-data: $(patsubst %, prepare-data-%, $(ALL_RUNS))
+
+{% for run in config.boomer_config.runs %}
+.PHONY: prepare-data-{{ run.id }}
+prepare-data-{{ run.id }}: $(BOOMER_INPUT_DIR)/{{ run.id }}/combined.ptable.tsv $(BOOMER_INPUT_DIR)/{{ run.id }}/prefix.yaml $(BOOMER_INPUT_DIR)/{{ run.id }}/symbiont-merged-all.owl 
+{% endfor %}
+
+.PHONY: boomer
+boomer: $(patsubst %, boomer-%, $(ALL_RUNS))
+
+.PHONY: clean
+clean:
+	rm -rf ./$(DATA)/*
+
+#####################
+### Dependencies ####
+#####################
+
+.PHONY: sssom
+sssom:
+	echo "skipping.."
+#	python3 -m pip install --upgrade pip setuptools && python3 -m pip install --upgrade --force-reinstall git+https://github.com/mapping-commons/sssom-py.git@curie_detection_patch
+#	python3 -m pip install --upgrade pip setuptools && python3 -m pip install --upgrade --force-reinstall sssom==0.3.7
+
+#####################
+## Ontologies #######
+#####################
+
+ONTOLOGIES= {% for ontology in ontologies %}{{ ontology.id }} {% endfor %}
+ONTOLOGY_PREFIXES= {% for ontology in ontologies %}{{ ontology.prefix }} {% endfor %}
+
+ALL_ONTOLOGIES=$(patsubst %, $(TMP_DIR)/%.owl, $(ONTOLOGIES))
+
+{% for ontology in ontologies %}
+$(TMP_DIR)/{{ ontology.id }}.owl: | $(TMP_DIR)/
+	wget {{ ontology.mirror_from|default('http://purl.obolibrary.org/obo/' ~ ontology.id ~ '.owl', true)}} -O $@
+{% endfor %}
+
+$(TMP_DIR)/symbiont-merged-all.owl: $(ALL_ONTOLOGIES)
+	robot merge $(addprefix -i , $^) --output $@
+
+$(BOOMER_INPUT_DIR)/%/symbiont-merged-all.owl: $(TMP_DIR)/symbiont-merged-all.owl
+	cp $< $@
+
+#####################
+## Mappings #########
+#####################
+
+ALL_MAPPINGS={% for mapping in mapping_registry.mapping_set_references %}$(TMP_DIR)/{{ mapping.local_name }} {% endfor %}
+
+{% for mapping in mapping_registry.mapping_set_references %}
+$(TMP_DIR)/{{ mapping.local_name }}: | $(TMP_DIR)/
+	wget {{ mapping.mirror_from|default(mapping.mapping_set_id) }} -O $@
+{% endfor %}
+
+$(BOOMER_INPUT_DIR)/%/combined.sssom.tsv $(BOOMER_INPUT_DIR)/%/prefix.yaml: $(ALL_MAPPINGS) | $(BOOMER_INPUT_DIR)/%/
```

Comment:
@hrshdhgd This make target needs fixing. Everything else I think works fine. Basically you can change all lines from `echo  "Harshad` to the end of the target. Basically you will have to rewrite gen_boomer_output in a way that it accepts the parameters i am passing in. Maybe add an Argparse or some such.

Reply:
Done! But boomer takes a long time and doesn't go through will all mappings. I'm not sure if something is wrong. I had to forcibly stop.